content/en/blog/general/lets-play-kyverno/index.md

```diff
@@ -0,0 +1,95 @@
+
+![Kyverno Playground: Context Panel](context.png)
+
+## Conclusion
+
+This article shows the story behind the Kyverno playground. How it started and where we are just three weeks later. We are really impressed with the initial feedback from our users and what we can already offer without having thought about it at first. We're excited to see how these tools evolve and offer in function. We hope to get more feedback to improve and develop it further.
```

Comment:
"This article explained the story behind the Kyverno Playground, how it started, and where we are just three weeks later."